AdyenActions/UI/View Controllers/QR Code/String+QRCode.swift

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+extension String {
+    
+    /// Generates a QRCode as UIImage representing the string
+    func generateQRCode() -> UIImage? {
+        guard let filter = CIFilter(name: "CIQRCodeGenerator") else {
+            return nil
+        }
+        
+        let data = self.data(using: String.Encoding.ascii)
+        filter.setValue(data, forKey: "inputMessage")
+        let transform = CGAffineTransform(scaleX: 3, y: 3)
```

Comment:
Can't we just set this as a view constraint instead of manipulating the image itself?